theforeman.org/pipelines/lib/foreman_infra.groovy

```diff
@@ -10,7 +10,7 @@ void git_clone_jenkins_jobs(args = [:]) {
     target_dir = args.target_dir ?: ''
 
     dir(target_dir) {
-        git url: 'https://github.com/theforeman/jenkins-jobs', poll: false
+        git url: 'https://github.com/evgeni/jenkins-jobs', branch: 'centos-ocp', poll: false
```

Comment:
this can go, I just needed it here so it pulls the ansible playbooks from my fork